docker-compose-dev-front.yml

```diff
@@ -0,0 +1,37 @@
+# Shanoir NG - Import, manage and share neuroimaging data
+# Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+# Contact us on https://project.inria.fr/shanoir/
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+
+version: '3'
+services:
+
+  nginx:
+    environment: 
+      - SHANOIR_DEV=TRUE
+    depends_on:
+      - "front-dev"
+
+  front-dev:
+    container_name: front-dev
+    build:
+      context: docker-compose-dev
```

Comment:
*context* here refers the the source directory for the docker build (i.e. the `docker-compose-dev/` directory in the source tree). You should keep `docker-compose` here, otherwise it will break the build.

This comment also applies to all images defined in docker-compose-dev.yml.